src/main/java/dal/fatih/todoly/Main.java

```diff
@@ -101,6 +131,13 @@ public static void filterTask() {
     }
 
     public static void main(String[] args) {
+        try {
+            inputTask = new ObjectInputStream(new FileInputStream(file));
+            tasks = (HashMap) inputTask.readObject();
+            inputTask.close();
+        } catch (Exception e) {
+            System.out.println(e);
+        }

```

Comment:
Maybe create a method and call it from here to have better readability of the code. The method can be called `loadTasksFromFile`